src/main/java/be/seeseemelk/mockbukkit/entity/LivingEntityMock.java

```diff
@@ -566,13 +566,13 @@ public boolean getCanPickupItems()
 	}
 
 	@Override
-	public boolean teleport(@NotNull Location location, PlayerTeleportEvent.@NotNull TeleportCause cause)
+	public boolean teleport(@NotNull Location location, PlayerTeleportEvent.@NotNull TeleportCause cause, boolean ignorePassengers, boolean dismount)
```

Comment:
Why was this changed?

Reply:
In order to maintain the health check when using the new Paper's API:
```java
// Redirects to teleport(Location, TeleportCause) that checks health, then uses teleport(Location, TeleportCause, boolean, boolean).
livingEntity.teleport(loc);
// Would skip the health check since this method is only implemented in the super class.
livingEntity.teleport(loc, TeleportCause.PLUGIN, true, true);
```
In fact, the `LivingEntity` class was not modified by PaperMC/Paper@5deafd1969b3c521af9cabb46822ecb01e6ce498, something that has been forgotten?

Reply:
Since it wasn't modified in Paper I feel as if it should not be modified here either to maintain behavior parody. If Paper ever fixes it then we can implement it here.